backend/README.md

```diff
@@ -1,33 +1,45 @@
-# Project Name
 
-## Description
-This code streams data from a device using the Lab Streaming Layer (LSL) protocol. It demonstrates how to connect to a data stream, pull samples, and print the output using Python.
+# Step-Time Biofeedback Backend
 
-## Requirements
+## Description
+This README provides instructions on how to install necessary dependencies, and run the tests for `Step_Time_Calculation.py`
 
-Before running this code, make sure you have the following installed on your machine:
+## Prerequisites
+Before running any code, make sure you have the following installed on your machine:
 
 ### Python Packages

```

Comment:
I think you can probably just put something here about needing to do a pip install -r requirements.txt and add the required libraries in the requirements.txt file. You could probably keep this as a just in case though but maybe add something to make sure it matches the version in the requirements.txt file.